hadoop-hdfs-project/hadoop-hdfs-httpfs/src/main/java/org/apache/hadoop/fs/http/client/HttpFSFileSystem.java

```diff
@@ -208,6 +210,9 @@ public static FILE_TYPE getType(FileStatus fileStatus) {
   public static final String XATTRNAMES_JSON = "XAttrNames";
   public static final String ECPOLICY_JSON = "ecPolicyObj";
   public static final String SYMLINK_JSON = "symlink";
+  public static final String CAPACITY_JSON="capacity";
```

Comment:
public static final String CAPACITY_JSON = "capacity";  A space is required after the equal sign.